app/config/settings_prod.py

```diff
@@ -262,3 +269,8 @@ def get_logging_config():
 # Timeout in seconds for call to external services, e.g. HTTP HEAD request to
 # data.geo.admin.ch/collection/item/asset to check if asset exists.
 EXTERNAL_SERVICE_TIMEOUT = 3
+
+# By default django_promtheus tracks the number of migrations
+# This causes troubles during linting, so we need to be able to
+# disable this during linting.
+PROMETHEUS_EXPORT_MIGRATIONS = os.getenv('PROMETHEUS_EXPORT_MIGRATIONS', True)
```

Comment:
Here we need to convert string to bool:

`PROMETHEUS_EXPORT_MIGRATIONS = strtobool(os.getenv('PROMETHEUS_EXPORT_MIGRATIONS', 'True'))`